extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -134,34 +134,81 @@ Metals effectively absorb all refracted light (light that isn't reflected), prev
 The metallic parameter of a glTF material effectively scales the `baseColor` of the material toward black while, at the same time scaling the F0 (reflectivity) value towards 1.0. This makes the material opaque for metallic values of 1.0 because transmitted light is attenuated out by absorption. Therefore, for a material with `metallicFactor=1.0`, the value of `transmissionFactor` doesn't matter.
 
 ## Transmission BTDF ##
-From the core [glTF BRDF](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-b-brdf-implementation), we have:
-
-*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub>
 
-*f*<sub>*diffuse*</sub> = (1 - *F*) * *diffuse*
+From the core [glTF BRDF](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-b-brdf-implementation), we have:
 
-*f*<sub>*specular*</sub> = *F* * *G* * *D* / (4 * dot(*N*, *L*) * dot(*N*, *V*)), where *F* is the Surface Reflection Ratio.
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = diffuse_brdf(color = baseColor),
+    layer = specular_brdf(α = roughness^2))
+```
 
 We will now add an additional term for the transmitted light:
 
-*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub> + *f*<sub>*transmission*</sub>
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = mix(
+      diffuse_brdf(baseColor),
+      specular_btdf(α = roughness^2) * baseColor,
+      transmission),
+    layer = specular_brdf(α = roughness^2))
+```
 
-*f*<sub>*transmission*</sub> = (1 - *F*) * *T* * *baseColor* * *D<sub>T</sub>* * *G<sub>T</sub>* / *(4 * abs(dot(N, L))* * *abs(dot(N, V)))*
+where `transmission` is the transmission percentage defined by this extension's `transmissionFactor` and `transmissionTexture` properties and `specular_btdf` is a bidirectional transmission distribution function (BTDF) based on microfacet theory. The microfacet distribution function is the same Trowbridge-Reitz model used by specular reflection except sampled along the view vector rather than the reflection. The *baseColor* factor causes the transmitted light to be tinted by the surface.
```

Comment:
It would be helpful to write out the definition of `specular_btdf` (using the D<sub>T</sub> G<sub>T</sub> notation).